Userland/Libraries/LibJS/AST.cpp

```diff
@@ -4962,9 +4962,9 @@ ModuleRequest::ModuleRequest(DeprecatedFlyString module_specifier_, Vector<Asser
     });
 }
 
-DeprecatedString const& SourceRange::filename() const
```

Comment:
Alternatively could store a copy of `code->filename().to_deprecated_string()` and return a reference to it.

Reply:
I think this is fine, ideally this would return a `String const&` in the future again resolving this issue.